components/uaepass-authenticator/src/main/java/org/wso2/carbon/identity/authenticator/uaepass/UAEPassAuthenticatorConstants.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.authenticator.uaepass;
+
+/**
+ * Includes all the constants variables used by the UAEPass authenticator.
+ */
+public class UAEPassAuthenticatorConstants {
+    public enum ErrorMessages {
+
+        USER_ID_NOT_FOUND_IN_ID_TOKEN_SENT_BY_FEDERATED_IDP("UAEPass-60001",
+                "Cannot find the userId from the id_token sent by the UAEPass IDP."),
+        RETRIEVING_MULTI_ATTRIBUTE_SEPARATOR_FAILED("UAEPass-65002",
+                "Error while retrieving multi attribute separator"),
+        EXECUTING_CLAIM_TRANSFORMATION_FOR_IDP_FAILED("UAEPass-65003",
+                "Error while executing claim transformation for UAEPass: %s");
+
+        private final String code;
+        private final String message;
+
+        /**
+         * Create an Error Message.
+         *
+         * @param code    Relevant error code.
+         * @param message Relevant error message.
+         */
+        ErrorMessages(String code, String message) {
+
+            this.code = code;
+            this.message = message;
+        }
+
+        /**
+         * To get the code of specific error.
+         *
+         * @return Error code.
+         */
+        public String getCode() {
+
+            return code;
+        }
+
+        /**
+         * To get the message of specific error.
+         *
+         * @return Error message.
+         */
+        public String getMessage() {
+
+            return message;
+        }
+
+        @Override
+        public String toString() {
+
+            return String.format("%s  - %s", code, message);
+        }
+    }
+
+    public static class UAEPassRuntimeConstants {
```

Comment:
Add a new line. Let's check and correct in all occurances

Reply:
Fixed with : [e41125306](https://github.com/wso2-extensions/identity-outbound-auth-uaepass/pull/1/commits/e41125306054f4b7345ef374ad89a9ab2fdfea52)